microprot/scripts/process_fasta.py

```diff
@@ -1,21 +1,22 @@
 import os
 import urllib
 import click
+import skbio
```

Comment:
I agree, but it seems that I need skbio in order to do type checking (line 150). Unless, I can import a only `skbio.sequence` and `skbio.io`?